app/helpers/members_helper.rb

```diff
@@ -68,4 +68,8 @@ def member_history_sentence(member)
     end.to_sentence
     text.html_safe + "."
   end
+
+  def display_name_ukr(name)
```

Comment:
I prefer, and it's common in Ruby, to use non-abbreviated method names so they're easier to read and understand. I also think it's helpful indicate what attribute this method is used to display.

So I'll change this method name to `ukrainian_party_display_name`.